Test2/Tests/HistoryPageTests.cs

```diff
@@ -0,0 +1,145 @@
+            historyPage.Open();
+           
+            Assert.AreEqual(expected, historyPage.LastResults);
+        }
+        
+        [Test]
+        public void ClearHistory()
+        {
+            HistoryPage historyPage = new HistoryPage(driver);
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+
+            calculatorPage.Open();
+            calculatorPage.Calculate("100000", "100", "365", "01", "January", "2022");
+            historyPage.Open();
+            historyPage.ClearButton.Click();
+
+            Assert.AreEqual(0, historyPage.Rows.Count);
+        }
+
+        [Test]
+        public void NumberRow()
+        {
+            HistoryPage historyPage = new HistoryPage(driver);
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+
+            historyPage.ClearButton.Click();
+            for (int i = 0; i < 10; i++)
+            {
+                calculatorPage.Open();
+                calculatorPage.Calculate("100000", "100", "365", "01", "January", "2022");
+            }
+            historyPage.Open();
+           
+            Assert.AreEqual(10, historyPage.Rows.Count);
+        }
+
+        [Test]
+        public void Lists()
+        {
+            new HistoryPage(driver).Open();
+
+            List<List<string>> tabelValues = new List<List<string>>();
+            List<IWebElement> Rows = driver.FindElements(By.XPath("//tr")).ToList();
+
+            foreach (IWebElement row in Rows)
+            {
+                List<string> RowValues = new List<string>();
+                List<IWebElement> cells = row.FindElements(By.XPath(".//td")).ToList();
+
+                foreach (IWebElement cell in cells)
+                {
+                    RowValues.Add(cell.Text);
+                }
+
+                tabelValues.Add(RowValues);
+            }
+        }
+
+        [Test]
+        public void List2()
+        {
+            new HistoryPage(driver).Open();
+            List<List<string>> tabelValues = new List<List<string>>();
+            int RowsCount = driver.FindElements(By.XPath("//tr")).Count();
+
+            for (int rowNumber = 0; rowNumber < RowsCount; rowNumber++)
+            {
+                List<string> rowValues = new List<string>();
+                for (int colNumber = 0; colNumber < 8; colNumber++)
+                {
+                    IWebElement cell = driver.FindElement(By.XPath($"//tr[{rowNumber+1}]//*[{colNumber+1}]"));
+                    string cellText = cell.Text;
+                    rowValues.Add(cellText);
+                }
+                tabelValues.Add(rowValues);
+            }
+        }
+
+        [Test]
+        public void CheckElements()
+        {
+            new HistoryPage(driver).Open();
+            HistoryPage historyPage = new HistoryPage(driver);
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+            List<List<string>> expected2 = new List<List<string>>() {
```

Comment:
Please rename this variable and add a line-break before `{`
```suggestion
            List<List<string>> expected = new List<List<string>>()
            {
```